src/04-modality-specific-files/01-magnetic-resonance-imaging-data.md

```diff
@@ -447,6 +447,135 @@ JSON example:
+```
+
+The complete ASL timeseries should be stored as a 4D NIfTI file in the original acquisition order, accompanied by two ancillary 
+files: `*_asl.json` and `*_aslcontext.tsv`. The raw BIDS NIfTIs should contain appropriately scaled data and no scaling factors 
+should be stored in the ancillary files or in the NIfTI header. 
+
+The `m0scan` can either be stored inside the 4D ASL time-series NIfTI file or as a separate NIfTI file, depending on whether it 
+was acquired within the ASL time-series or as a separate scan. These and other M0 options are specified in the required M0 field. 
+It can also be stored under `fmap/sub-<label>[_ses-<label>][_acq-<label>][_ce-<label>]_dir-<label>[_run-<index>]_m0scan.nii[.gz]`, 
+when the pepolar approach is used.
+
+The `*_aslcontext.tsv` lists the volume_type for each volume in a single column, in the `*_asl.nii[.gz]`, corresponding to the volume_types specified in the table below, based 
+on DICOM Tag (0018,9257) `ASL Context`. Note that the volume_types `control` and  `label` within BIDS do not discriminate between types of labeling, but only serve to specify 
+the magnetization state of the blood and thus the ASL subtraction order.
+
+| **volume_type** | **Definition**                                                                                                                                                                         |
+|-----------------|----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| control         | The control image is acquired in the exact same way as the label image, except that the magnetization of the blood flowing into the imaging region has not been inverted.              |
+| label           | The label image is acquired in the exact same way as the control image, except that the blood magnetization flowing into the imaging region has been inverted.                         |
+| m0scan          | The M0 image is a calibration image, used to estimate the equilibruim magnetization of blood.                                                                                          |
+| deltam          | The deltaM image is a perfusion-weighted image, obtained by the subtraction of `control` - `label`.                                                                                    |
+| cbf             | The cerebral blood flow (CBF) image is produced by dividing the deltaM by the M0, quantified into `mL/100g/min` (See also [doi:10.1002/mrm.25197](https://doi.org/10.1002/mrm.25197)). |
+
+If the raw `control` and `label` images are not available, their derivative `deltam` should be stored as raw data instead. If the 
+`deltam` is not available, `cbf` should be stored as raw data instead. When `cbf` is stored as raw data, its units need to be 
+specified in the `*_asl.json` as well.
```

Comment:
As mentioned in https://github.com/bids-standard/bids-specification/pull/652/files#r516908561, this description can be confusing. I think it's important to explicitly link `deltam` and `cbf` as volume types (rather than modalities or suffixes) to the `asl.nii.gz` file. A good way to do that would be to replace "raw data" (the whole BIDS dataset is essentially "raw data") with "`_asl.nii.gz`".

E.g., "their derivative `deltam` should be stored in the `_asl.nii.gz` file instead."

Reply:
Yes this makes sense, thanks!